### ISSUE TITLE
UX/accessibility Issue - Cannot reach the Linked Accounts with a screen reader navigation 

### ISSUE THREAD

**Issue by PereViki** [topcoder-platform/community-app#1524]
**Describe the bug**
If I try to browse the site with a screen reader (VoiceOver) I cannot add linked accounts as they are not reachable with the navigation of the screen reader.

**To Reproduce**

1. Turn on your screen reader (VoiceOver)
2. Go to 'https://www.topcoder-dev.com/settings/account'
3. Click on 'Linked Account'
4. Try to navigate with the screen reader to any of the linked account icons.
5. See error

**Expected behavior**
One should be able to add a linked account even with a screen reader.

**Screenshots**
https://www.screencast.com/t/0ezw1VAPy

**Desktop: MacBook Pro**
 - OS: Mac OS High Sierra
 - Browser: Chrome
 - Version: 69.0.3497.92

@ThomasKranitsas
